broker/src/main/java/io/zeebe/broker/system/configuration/EnvironmentConstants.java

```diff
@@ -9,17 +9,5 @@
 
 public final class EnvironmentConstants {
 
-  public static final String ENV_NODE_ID = "ZEEBE_NODE_ID";
-  public static final String ENV_HOST = "ZEEBE_HOST";
-  public static final String ENV_ADVERTISED_HOST = "ZEEBE_ADVERTISED_HOST";
-  public static final String ENV_PORT_OFFSET = "ZEEBE_PORT_OFFSET";
-  public static final String ENV_INITIAL_CONTACT_POINTS = "ZEEBE_CONTACT_POINTS";
-  public static final String ENV_DIRECTORIES = "ZEEBE_DIRECTORIES";
-  public static final String ENV_PARTITIONS_COUNT = "ZEEBE_PARTITIONS_COUNT";
-  public static final String ENV_REPLICATION_FACTOR = "ZEEBE_REPLICATION_FACTOR";
-  public static final String ENV_CLUSTER_SIZE = "ZEEBE_CLUSTER_SIZE";
-  public static final String ENV_CLUSTER_NAME = "ZEEBE_CLUSTER_NAME";
-  public static final String ENV_EMBED_GATEWAY = "ZEEBE_EMBED_GATEWAY";
   public static final String ENV_DEBUG_EXPORTER = "ZEEBE_DEBUG";
```

Comment:
I now moved it into `BrokerCfg`

I kept it there, initially,  mostly because I wanted to wait till I implement the "strict mode" (which I haven't implemented). 
I am not a fan of constant-only-classes. But I am a fan of having a class where I find several developer flags/feature toggles in one place. But this can wait to another day.